core/authenticate/service.go

```diff
@@ -316,6 +385,208 @@ func (s Service) applyMailOTP(ctx context.Context, request RegistrationFinishReq
 	}, nil
 }
 
+func (s Service) startPassKeyRegisterMethod(ctx context.Context, flow *Flow) (*RegistrationStartResponse, error) {
+	newPassKeyUser := strategy.NewPassKeyUser(flow.Email)
+	options, session, err := s.webAuth.BeginRegistration(newPassKeyUser)
+	if err != nil {
+		return nil, err
+	}
+
+	session.Challenge = base64.RawURLEncoding.EncodeToString([]byte(session.Challenge))
```

Comment:
add a comment here why we are doing this instead of directly using it?

Reply:
done